update: More detailed Session description (#2519)
privacyguides/privacyguides.org: docs/real-time-communication.md

</div>

> Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. Open groups have no restriction on the number of members, they are hosted on federated servers operated by the communities themselves, and moderation policies are determined by each individual community. However, they are not as private as one-on-one chats or closed groups. 

Reviewer: ```suggestion
Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [host](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join an open group which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.
```

</div>

> Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [host](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join an open group which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.

Reviewer 2: ```suggestion
Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [set up](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join open groups which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.
```

</div>

Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [host](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join an open group which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.

> 

Reviewer 3: ```suggestion
```
